tests/test_x52f.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from streaming_data_types.status_x5f2 import serialise_x5f2, deserialise_x5f2
+
+
+class TestEncoder(object):
+
+    def test_serialises_and_deserialises_x5f2_message_correctly(self):
+        #     """
+        #     Round-trip to check what we serialise is what we get back.
+        #     """
+
+        original_entry = {
+            "software_name" : "nicos/test",
+            "software_version" : "1.0",
```

Comment:
We are using MAJOR.MINOR.PATCH versioning, so may as well make the example match:
```suggestion
            "software_version" : "1.0.0",
```